bugbug/model.py

```diff
@@ -91,19 +91,29 @@ def classification_report_imbalanced_values(
     return result
 
 
-def get_labeled_confusion_matrix(y_test, y_pred, labels):
-    confusion_matrix = metrics.confusion_matrix(y_test, y_pred, labels=labels)
-    confusion_matrix_table = confusion_matrix.tolist()
-    confusion_matrix_header = []
-    for i in range(len(confusion_matrix_table)):
-        confusion_matrix_header.append(f"{labels[i]} (Predicted)")
-    for i in range(len(confusion_matrix_table)):
-        confusion_matrix_table[i].insert(0, f"{labels[i]} (Actual)")
-    labeled_confusion_matrix = tabulate(
-        confusion_matrix_table, headers=confusion_matrix_header, tablefmt="fancy_grid"
-    )
-
-    return labeled_confusion_matrix
+def print_labeled_confusion_matrix(y_test, y_pred, labels, is_multilabel=False):
```

Comment:
You could pass the confusion matrix directly, since you're already calculating it in train, so you can avoid to recalculate it here